eas.json

```
@@ -8,11 +8,11 @@
     "base": {
       "android": {
         "image": "latest",
-        "node": "20.17.0"
+        "node": "20.19.0"
       },
       "ios": {
         "image": "latest",
-        "node": "20.17.0"
+        "node": "20.19.0"
```

Should match exact version used in dev + CI

```suggestion
        "node": "20.19.5"
},
"ios": {
"image": "latest",
        "node": "20.19.5"
```

---

https://github.com/digidem/comapeo-mobile/pull/1559/commits/a7e85be5e4ab910297b4983ef840b711f53fb20a